profile/repositories-list.md

```diff
@@ -0,0 +1,147 @@
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [Wallet UI App (Android)](https://github.com/eu-digital-identity-wallet/eudi-app-android-wallet-ui) | Implementation of wallet UI app for Android. Currently, it also includes Demo App, demonstrating the following capabilities: Proximity presentation, Same Device Online Authentication and issuing of PID and mDL.	 |
+| [Wallet UI App (iOS)](https://github.com/eu-digital-identity-wallet/eudi-app-ios-wallet-ui) | Implementation of wallet UI app for iOS. Currently, it also includes Demo App, demonstrating the following capabilities: Proximity presentation, and Same Device Online Presentation and issuing of PID and mDL.| 
+
+</br>
+
+## Verifier Apps and Services
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [Web Verifier](https://github.com/eu-digital-identity-wallet/eudi-web-verifier)               | Demo Web Verifier UI application (Frontend) that acts as a Verifier/RP trusted end-point. Available at [https://verifier.eudiw.dev](https://verifier.eudiw.dev)  |  
+| [Restful API (web-services)](https://github.com/eu-digital-identity-wallet/eudi-srv-web-verifier-endpoint-23220-4-kt)             | Demo Web Verifier application (Backend Restful service) that acts as a Verifier/RP trusted end-point.| 
+
+</br>
+
+## Issuing Apps and Services
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [OpenId4VCI issuer (Python)](https://github.com/eu-digital-identity-wallet/eudi-srv-web-issuing-eudiw-py)               | An implementation of a credential issuing service, according to OpenId4VCI (draft 15), in Python. Available at https://issuer.eudiw.dev/  |  
+| [OpenId4VCI issuer (Kotlin)](https://github.com/eu-digital-identity-wallet/eudi-srv-pid-issuer)             | An implementation of a credential issuing service, according to OpenId4VCI (draft 15), in JVM Kotlin. Available at  https://issuer-backend.eudiw.dev/ | 
+
+
+</br>
+
+## Signing Apps and Services
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [Trust Provider Signer](https://github.com/eu-digital-identity-wallet/eudi-srv-web-trustprovider-signer-java)               | TrustProvider Signer is a remote signing service provider and client. The trustprovider signer can be accessed using the EUDI Wallet through OID4VP presentation of the PID. Available [here](https://trustprovider.signer.eudiw.dev/tester)  |  
```

Comment:
'TrustProvider Signer' → be consistent ('Trust Provider Signer' or 'TrustProvider Signer').

Reply:
'The trustprovider signer' → capitalisation: 'The Trust Provider Signer'.